service/history/api/queryworkflow/api.go

```diff
@@ -364,3 +366,16 @@ func queryDirectlyThroughMatching(
 			QueryRejected: matchingResp.GetQueryRejected(),
 		}}, err
 }
+
+func GetWorkflowVersioningInfoMatchingTask(msResp *historyservice.GetMutableStateResponse) *deploymentpb.WorkflowVersioningInfo {
```

Comment:
Everything besides `Invoke` in `api` dir must be unexported.

Reply:
removed this.